packages/cli/src/runAndroid/runAndroid.js

```diff
@@ -155,12 +155,21 @@ function buildApk(gradlew) {
   }
 }
 
-function tryInstallAppOnDevice(args, device) {
+function tryInstallAppOnDevice(args, adbPath, device) {
   try {
     // "app" is usually the default value for Android apps with only 1 app
     const appFolder = args.appFolder || 'app';
-    const pathToApk = `${appFolder}/build/outputs/apk/${appFolder}-debug.apk`;
-    const adbPath = getAdbPath();
+    const variant = (args.variant || 'debug').toLowerCase();
```

Comment:
I think we can define "default: debug" at the bottom, where we have CLI arguments defined. What do you think?

Reply:
@grabbou I think the way you said is good.